packages/sdk/src/sendPanic.ts

```diff
@@ -93,6 +90,7 @@ export async function sendPanic(error: RustPanic, cliVersion: string, engineVers
     return id
   } catch (e) {
     debug(e)
+    throw e
```

Comment:
This propagates the exception to `handlePanic` as required by https://github.com/prisma/prisma/issues/12739